client/app/controllers/projects.js

```diff
@@ -1,41 +1,84 @@
 import Controller from '@ember/controller';
+import { sort } from '@ember/object/computed';
+
+// TODO: Get theses codes from a Constants service
```

Comment:
My only objection to using a service would be that services are stateful objects that get initialized on app bootup, and they would be overkill for this because these constants are static. We might just keep them in a util file somewhere and import where it's needed. Thoughts?